trans.py

```diff
@@ -53,7 +53,7 @@
 os.system('crowdin-cli --identity=keys/crowdin.yaml download')
 
 os.chdir(os.path.dirname(os.path.realpath(__file__)))
-root = 'electron/locale/'
+root = 'electron/src/locale/'
 
 
 def get_locale(filename):
```

Comment:
This needs to be adjusted to write the TS files before merging the PR. We can move to the JS approach later on but the code needs to continue to work.